src/main/engine/SceneObject/headers/SceneObject.hpp

```diff
@@ -60,7 +60,7 @@ class SceneObject {
 
     const string objectName;
     GLfloat scale;
-    GLuint programId;
+    GLuint programId_;
```

Comment:
Curious about the renaming here. Is there a conflict with something? Style?

Reply:
I've been kind of lazy about this, but it's a styling I'd like to adopt. I'd like to have an underscore suffix (_) on private member variables names so it's easier to tell at a glance if something is a class or scope level variable.

Reply:
Oh, makes sense. Every time I look at it I think of "__foo" syntax of platform level stuff.